Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Build stage: Install python dependencies
 # ===
-FROM ubuntu:focal AS python-dependencies
-RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+FROM ubuntu:bionic AS python-dependencies
```

Comment:
I was wondering why we changed from focal (20.04) to bionic (18.04) as it's a more up to date version?

Reply:
Pilbox pulls an old version of Pillow that fails to install under python3.8(focal).